spotless.license-java

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
I used to add the filename in the license header here. Can we do the same... and is it a good idea?

Reply:
Doesn't seem like spotless support any such placeholder as of now. Maybe we can create a feature request for them.